pkg/apis/eventing/v1alpha1/cluster_provisioner_types.go

```diff
@@ -67,42 +70,12 @@ type ClusterProvisionerSpec struct {
 	Reconciles metav1.GroupKind `json:"reconciles"`
 }
 
-type ClusterProvisionerConditionType string
-
-const (
-	// ClusterProvisionerConditionReady specifies that the resource is ready.
-	ClusterProvisionerConditionReady ClusterProvisionerConditionType = "Ready"
-)
-
-// ClusterProvisionerConditionStatus describes the state of this resource at a point in time.
-type ClusterProvisionerConditionStatus struct {
-	// Type of condition.
-	// +required
-	Type ClusterProvisionerConditionType `json:"type"`
-
-	// Status of the condition, one of True, False, Unknown.
-	// +required
-	Status corev1.ConditionStatus `json:"status"`
-
-	// LastTransitionTime is the last time the condition transitioned from one status to another.
-	// We use VolatileTime in place of metav1.Time to exclude this from creating equality.Semantic
-	// differences (all other things held constant).
-	// +optional
-	LastTransitionTime apis.VolatileTime `json:"lastTransitionTime,omitempty" description:"last time the condition transit from one status to another"`
-
-	// The reason for the condition's last transition.
-	// +optional
-	Reason string `json:"reason,omitempty"`
-
-	// A human readable message indicating details about the transition.
-	// +optional
-	Message string `json:"message,omitempty"`
-}
+var cProvCondSet = duck.NewLivingConditionSet()
 
 // ClusterProvisionerStatus is the status for a ClusterProvisioner resource
 type ClusterProvisionerStatus struct {
 	// Conditions holds the state of a cluster provisioner at a point in time.
-	Conditions []ClusterProvisionerConditionStatus `json:"conditions,omitempty"`
+	Conditions duck.Conditions `json:"conditions,omitempty"`
```

Comment:
Why doesn't this field have the `patch` tags that `SubscriptionStatus.Conditions` has?